pyconcz/templates/pages/board_game_night.html

```diff
@@ -19,31 +19,29 @@ <h1>Board Game Night</h1>
                 <h2>Rules of Board Game Night</h2>
                 <ol>
                     <li>You do talk about Board Game Night.</li>
-                    <li>You <em style=": uppercase;">do</em> talk about Board Game Night.</li>
+                    <li>You <strong class="text-uppercase">do</strong> talk about Board Game Night.</li>
                     <li>Anyone can play.</li>
                     <li>More games at a&nbsp;time.</li>
                     <li>Shirts all time, shoes optional.</li>
                     <li>Games will go on as long as they have to or until 1 am.</li>
-                    <li>If this is your first night at Board Game Night, you HAVE to play. (Unless you don’t want to.)</li>
+                    <li>If this is your first night at Board Game Night, you <strong class="text-uppercase">have</strong> to play. (Unless you don’t want to.)</li>
                     <li>You can bring your own board games.</li>
-                    <li>PyCon CZ <a href="{% url 'about_code' %}">CoC</a> applies to Board Game Night.</li>
+                    <li>PyCon&nbsp;CZ <a href="{% url 'about_code' %}">CoC</a> applies to Board Game Night.</li>
                 </ol>
             </div>
             <div class="col-md">
                 <h2>Also good to know</h2>
                 <ul>
-                    <li>We will bring some games (DiXit, Pandemic, Steampunk Rally, Vast, Tzolk'in, Ubongo, Roll for the Galaxy, Robo Rally, 5 Minute Dungeon, Smallworld, Patchwork, Exploding Kittens, Hanabi, Carcassonne and more) but you’re welcome to bring your own as well!</li>
```

Comment:
I can bring the same games again this year. Probably some more, but I'll not be able to check until about May 14.

Reply:
Ok. Let's not list any and add them later?

Reply:
Let's publish what we know and add more later. We can even suggest making pull requests to this template with games ppl will bring 🤔😁

Reply:
@hroncok can we publish what we have and update later?

Reply:
I consider it silly to list 2 games (would rather not list anything now), but use your best judgement.